docs/outlook/autolaunch.md

```diff
@@ -399,6 +400,7 @@ As you develop your event-based add-in, you may need to troubleshoot issues, suc
 - Ensure that the following conditions are met in your add-in's manifest.
   - Verify that your add-in's source file location URL is publicly available and isn't blocked by a firewall. This URL is specified in your manifest's [SourceLocation element](/javascript/api/manifest/sourcelocation).
   - Verify that the **\<Runtimes\>** element correctly references the HTML or JavaScript file containing the event handlers. Outlook on Windows uses the JavaScript file during runtime, while Outlook on the web and on new Mac UI use the HTML file. For an example of how this is configured in the manifest, see [Configure the manifest](#configure-the-manifest).
+- An event-based add-in that runs in Outlook on Windows only supports a single JavaScript in the add-in project. You must bundle your JavaScript code into a single file.
```

Comment:
I'm unclear what "a single JavaScript" means. Can you clarify?

```suggestion
- An event-based add-in that runs in Outlook on Windows only supports a single JavaScript reference in the add-in project. You must bundle your JavaScript code into a single file.
```

Reply:
Thanks for catching that. That should say "single JavaScript file".